packages/core/src/components/radio-group/radio-group.tsx

```diff
@@ -22,7 +22,7 @@ export class RadioGroup implements ComponentDidLoad, RadioGroupInput {
   /*
    * If true, the user cannot interact with the radio group. Default false.
    */
-  @Prop({ mutable: true }) disabled = false;
```

Comment:
What does this change do? Better yet, can you point me to some docs on what that does and I will just read those.